### ISSUE TITLE
Highlight in pop-up menus not visible

### ISSUE THREAD

**Issue by JanPokorny** [ZYinMD/sepia#3]
Hello, it seems like this theme doesn't style the highlighted item in pop-up menus -- like the theme selection menu, for example ("Sepia" is selected, but you can't see that): 
![obrazek](https://user-images.githubusercontent.com/4580066/132338944-cd668c97-bfd5-44fd-893d-15987c4fbf9d.png)

Compare with other theme:
![obrazek](https://user-images.githubusercontent.com/4580066/132339013-8b28dbf2-e7e9-4434-aa78-e86093aadbfc.png)

This is specific to the keyboard-higlighted item -- hovering with mouse works fine.

**Comment by JanPokorny** [comment]
Workaround (put in `settings.json`):

```json
"workbench.colorCustomizations": {
    "[Sepia]": {
        "list.activeSelectionBackground": "#D4C8AC"
    }
}
```

**Comment by JanPokorny** [comment]
Merging #4 should fix this

**Comment by ZYinMD** [comment]
Merged, thanks very much. My github email notification was not turned on for new issues submitted to my repos (I think it's off by default for everybody) so I never saw your issue and your PR, until I found it by pure chance today. Sorry for the late reply.

Merging PR is one thing, publishing the new change to the vscode extension marketplace is another - I believe I just did that too, by running the deploy script I made 2 years ago, which I no longer remember how it works, but it seems to have succeeded.
